examples/typescript-plugin/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typescript-plugin",
+  "private": true,
+  "version": "0.1.0",
+  "description": "A Strapi plugin written in Typescript",
+  "license": "MIT",
+  "strapi": {
+    "displayName": "Typescript Plugin",
+    "name": "typescript-plugin",
+    "icon": "plug",
+    "description": "A Strapi plugin written in Typescript",
+    "kind": "plugin"
+  },
+  "scripts": {
+    "develop": "tsc -p tsconfig.json -w",
```

Comment:
should we name this watch or build:watch instead ?

Reply:
We can indeed